aten/src/ATen/core/dispatch/DispatchKeyExtractor.h

```diff
@@ -192,22 +217,23 @@ struct TORCH_API DispatchKeyExtractor final {
 
 private:
   static c10::utils::bitset makeBitsetForDispatchArgs(const FunctionSchema& schema) {
-    TORCH_CHECK(schema.arguments().size() <= c10::utils::bitset::NUM_BITS(),
-        "The function schema has ", schema.arguments().size(),
-        " arguments but this PyTorch build only supports ", c10::utils::bitset::NUM_BITS());
-    c10::utils::bitset dispatch_arg_indices_reverse;
-    for (const auto index : c10::irange(schema.arguments().size())) {
-      if (schema.arguments()[index].type()->isSubtypeOf(*TensorType::get()) ||
-          schema.arguments()[index].type()->isSubtypeOf(
-              *ListType::ofTensors()) ||
-          schema.arguments()[index].type()->isSubtypeOf(
-              *ListType::ofOptionalTensors()) ||
-          schema.arguments()[index].type()->isSubtypeOf(
-              *OptionalType::ofTensor())) {
-        dispatch_arg_indices_reverse.set(schema.arguments().size() - 1 - index);
+    if (schema.arguments().size() <= c10::utils::bitset::NUM_BITS()) {
```

Comment:
It might be worth noting that if we have an op with exactly `NUM_BITS` number of args where every argument is a tensor, won't the bitset now be ambiguous (it'll become the sentinel value)?

I guess this is ok though, because we when see the sentinel value at runtime we will just do a slower loop over all of the arguments, which is still correct (so we are not changing behavior)

Reply:
Yup, exactly!